admin_manual/configuration/server/background_jobs_configuration.rst

```diff
@@ -119,17 +116,6 @@ A number of existing background jobs are available to be run just for specific t
 
 While not exhaustive, these include:
 
-CleanupChunks
-^^^^^^^^^^^^^
-
-The CleanupChunks job, contained in ``OCA\\DAV\\Command\\CleanupChunks``, will clean up outdated chunks (uploaded files) more than a certain number of days old.
```

Comment:
we need to keep the command and mention that it needs to be added to cron. the only detail is that there is no matching background job to delete from the oc_jobs table

Reply:
Oh, ok. Sorry, I misunderstood.